ecosystem/lava-sdk/proto/lavanet/lava/pairing/query.proto

```diff
@@ -235,6 +235,7 @@ message QueryEffectivePolicyResponse {
 message QuerySdkPairingResponse {
   QueryGetPairingResponse pairing = 1;
```

Comment:
is this merged with main?, we dont push the lava-sdk/proto files as they just point to lava/proto should be in the gitignore in main

Reply:
yes, this is in main now, should I remove ecosystem/lava-sdk/proto folder from git?

Reply:
yes please also add ecosystem/lava-sdk/proto/lavanet to gitignore in a commit after didn't see its still in main repo